NEWS.md

```diff
@@ -9,10 +9,86 @@ ClamAV 0.105.0 includes the following improvements and changes.
 
 ### New Requirements
 
+- Starting with ClamAV v0.105, the Rust toolchain is required to compile ClamAV.
+
+  You can install the Rust toolchain for your development environment by
+  following the instructions on the [`rustup`](https://rustup.rs/) website.
+  Some binary package distributions do provide relatively up to date packages of
```

Comment:
"up-to-date" (you did it below)

You could also say "current" or "updated" if you want to mix it up.